cmd/zc_traverser_list.go

```diff
@@ -1,27 +1,32 @@
 package cmd
 
 import (
-	"bufio"
+	"context"
 	"fmt"
-	"github.com/Azure/azure-storage-azcopy/common"
-	"io"
 	"net/url"
+
+	"github.com/Azure/azure-storage-azcopy/common"
 )
 
 // a meta traverser that goes through a list of paths (potentially directory entities) and scans them one by one
 // behaves like a single traverser (basically a "traverser of traverser")
 type listTraverser struct {
-	listReader              io.ReadCloser
+	listReader              chan string
 	childTraverserGenerator childTraverserGenerator
 }
 
 type childTraverserGenerator func(childPath string) (resourceTraverser, error)
 
+func (l *listTraverser) isDirectory(isDest bool) bool {
```

Comment:
there's no impact for this returning false, right? since the listTraverser is a meta-traverser and does not represent a single resource dir/file, please leave a comment to explain accordingly

Reply:
Nope, no impact, note has been made